### PR TITLE
[Enhancement] (nereids)implement showProcessListCommand in nereids

### LINKED ISSUE #42729: [Enhancement] (nereids)implement showProcessListCommand in nereids
showProcessList command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following showProcessList command in nereids:
unsupportedshowProcessListStatement
``` SHOW FULL? PROCESSLIST                                                        ```

### PR DESCRIPTION
**Pull request by msridhar78**
Issue Number: close #42729 

implement showProcessListCommand in nereids

Regression Test results:
Changed  Session  Variables:
VarName                              |  CurrentValue                |  DefaultValue
----------------------|---------------------|-------------
character_set_results  |  NULL                                |  utf8mb4          
sql_mode                            |  STRICT_TRANS_TABLES  |                          
variable_version            |  0                                      |  300                  

2024-11-27 08:15:10.972 INFO [suite-thread-1] (Suite.groovy:1316) - Execute tag: cmd, sql: SHOW FULL PROCESSLIST
2024-11-27 08:15:10.981 INFO [suite-thread-1] (ScriptContext.groovy:120) - Run test_show_process_list_command in /root/doris/doris/regression-test/suites/nereids_p0/show/test_show_process_list_command.groovy succeed
2024-11-27 08:15:31.942 INFO [main] (RegressionTest.groovy:281) - Start to run single scripts
2024-11-27 08:16:57.142 INFO [main] (RegressionTest.groovy:410) - Success suites:
/root/doris/doris/regression-test/suites/nereids_p0/show/test_show_process_list_command.groovy: group=query,process_list,p0, name=test_show_process_list_command
2024-11-27 08:16:57.145 INFO [main] (RegressionTest.groovy:492) - All suites success.

### CODE REVIEW

**Comment by msridhar78** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowProcessListCommand.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import com.google.common.collect.Lists;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+/**
+ * Represents the command for SHOW PROCESSLIST
+ */
+public class ShowProcessListCommand extends Command implements NoForward {
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowProcessListCommand.java:48]:
```suggestion
public class ShowProcessListCommand extends ShowCommand {
```

**Comment by starocean999** [comment]
please add test case

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowProcessListCommand.java (hunk 1 above)
Comment (msridhar78) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowProcessListCommand.java]: done - please review

**Comment by starocean999** [comment]
please rebase to resolve conflicts

**Comment by msridhar78** [comment]
run buildall

**Review by LiBinfeng-01 — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**